<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="1,0,2,0,4">
      <item>
       <layout class="QVBoxLayout" name="infoLayout">
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="softNameLabel">
          <property name="text">
           <string>Astra Portal</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sensorInfoLabel">
          <property name="text">
           <string>Sensor info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sensorNameLabel">
          <property name="text">
           <string>Sensor name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serialLabel">
          <property name="text">
           <string>Serial number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="overlayCheckBox">
          <property name="text">
           <string>Overlay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="registrationCheckBox">
          <property name="text">
           <string>Hardware registration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorInfoLabel">
          <property name="text">
           <string>Color stream</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="colorMirrorCheckBox">
          <property name="text">
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="awbCheckBox">
          <property name="text">
           <string>Auto white balance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="aeCheckBox">
          <property name="text">
           <string>Auto exposure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="depthInfoLabel">
          <property name="text">
           <string>Depth stream</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="depthMirrorCheckBox">
          <property name="text">
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pointerModeCheckBox">
          <property name="text">
           <string>Point mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="pointcloudInfoLabel">
            <property name="text">
             <string>PointCloud</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showAxisCheckBox">
            <property name="text">
             <string>Show axis</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useRgbCheckBox">
            <property name="text">
             <string>Use rgb component</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useOwnParamCheckBox">
            <property name="text">
             <string>Use own camera param</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useExternalParamCheckBox">
            <property name="text">
             <string>Use external camera param</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePlyButton">
            <property name="text">
             <string>Save ply</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multiPointcloudButton">
            <property name="text">
             <string>MultiCloudPoint</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="layoutSplitLine1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="imageLayout" stretch="0,0,0">
        <item>
         <widget class="QLabel" name="colorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="imageSplitLine">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DepthLabel" name="depthLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="layoutSplitLine2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PointcloudGLWidget" name="pointCloudGLWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PointcloudGLWidget</class>
   <extends>QWidget</extends>
   <header>orbbec_pointcloud_gl_widget.h</header>
  </customwidget>
  <customwidget>
   <class>DepthLabel</class>
   <extends>QLabel</extends>
   <header location="global">orbbec_depth_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
